common-arcana.lic

```diff
@@ -83,6 +83,13 @@ module DRCA
     fix_standing
   end
 
+  def use_glyph(name)
```

Comment:
If this will return true/false (a 'predicate'), the Ruby convention is to have a `?` at the end of the function name (so `use_glyph?`).

Reply:
OK, will add.